### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by cdown** [dannyedel/dspdfviewer#228]
Hi there -- thanks for the very useful project which I often use for presentations. I notice that the latest version released is very old and doesn't build with modern Qt5 due to deprecation warnings (and -Werror). Maybe it's time for a new release? :-)

**Comment by cdown** [comment]
Current HEAD also fails to build, by the way:

```
/usr/include/boost/math/tools/config.hpp:23:6: warning: #warning "The minimum language standard to use Boost.Math will be C++14 starting in July 2023 (Boost 1.82 release)" [-Wcpp]
   23 | #    warning "The minimum language standard to use Boost.Math will be C++14 starting in July 2023 (Boost 1.82 release)"
      |      ^~~~~~~
```

**Comment by dannyedel** [comment]
Hi Chris,

thanks for letting me know.  Should be easy enough to fix.

I'll disable -Werror by default, because it's done more harm than good by failing distributions' build scripts.  I (and anyone else who wants to) can just turn it on via a CMake option.

With that said, I'll try to make it build on a current debian unstable without build-time warnings, let's hope that doesn't break builds on the older systems.

I'll post here once I have something ready to test.

**Comment by cdown** [comment]
Thank you! Much appreciated -- giving presentations is a large part of my job, and I give all of my presentations via dspdfviewer, so thank you very much!

**Comment by dannyedel** [comment]
I can't reproduce your exact error message, likely you have a slightly different system than me with an even more recent Qt/boost library.

Can you try building the "fix-master-build" branch (see #229) on your machine and tell me if that change works for you?

**Comment by cdown** [comment]
7c0b70b25ea8af25c58ba541865c22999cdcea95 builds great with the following versions -- thanks!

```
% pacman -Q | grep -e boost -e qt
boost 1.81.0-3
boost-libs 1.81.0-3
poppler-qt5 23.03.0-1
qt5-base 5.15.9+kde+r151-1
qt5-tools 5.15.9+kde+r1-1
qt5-translations 5.15.9-1
```
